action.yaml

```diff
@@ -85,9 +89,21 @@ runs:
           echo "==# Skipping S3 artefact deletion: no release identifier provided";
           exit 1
         fi
+        
+        s3_cut_position=0
+        if [[ "${s3_object_deletion_type}" == "file" ]]; then
+          echo "The target S3 artifact for deletion is a file"
+          s3_cut_position=4
+        elif [[ "${s3_object_deletion_type}" == "folder" ]]; then
+          echo "The target S3 artifact for deletion is a folder"
+          s3_cut_position=3
+        else
+          echo "Invalid s3-object-deletion-type: '${s3_object_deletion_type}'. The value must be either 'file' or 'folder'"
+          exit 1
+        fi
 
         s3_app_url="s3://${bucket_name}/${object_key_prefix}"
-        packages="$( aws s3 ls "${s3_app_url}/" | grep -- "${release_identifier}" | tr -s ' ' | cut -d ' ' -f 4 || true )"
+        packages="$( aws s3 ls "${s3_app_url}/" | grep -- "${release_identifier}" | tr -s ' ' | cut -d ' ' -f "${s3_cut_position}" || true )"
```

Comment:
I guess if we use `awk '{print $NF}'` instead of `cut` then there is no need to define that condition
https://www.gnu.org/software/gawk/manual/html_node/Fields.html